### ISSUE TITLE
Native libraries of secp256k1 and alt bn 128 cannot be disabled

### ISSUE THREAD

**Issue by daniel-iobuilders** [hyperledger/besu#2150]
Both `SECP256K1.class` and `AltBN128AddPrecompiledContract.class` have a Java implementation and another one using native libraries. The native library implementations are enabled by default in both cases. `BesuCommand` has the options `--Xsecp256k1-native-enabled` and `--Xaltbn128-native-enabled` which can be set to `false` to disable the native libraries and use the Java implementations. 

But right now the disabling of the libraries does not work:

`SECP256K1.class` and `AltBN128AddPrecompiledContract.class` have a boolean flag to enable or disable the native libraries, which is initialized with `true`:
```
private boolean useNative = true;
```

In `BesuCommand` the handling of the options looks like this:
```
  private void configureNativeLibs() {
    if (unstableNativeLibraryOptions.getNativeAltbn128()) {
      AbstractAltBnPrecompiledContract.enableNative();
    }
    if (unstableNativeLibraryOptions.getNativeSecp256k1()) {
      SignatureAlgorithmFactory.getInstance().enableNative();
    }
  }
```

`unstableNativeLibraryOptions.getNativeAltbn128()` and `unstableNativeLibraryOptions.getNativeSecp256k1()` return `true` by default and `false` when one of the corresponding options is set to `false`.

If one of the options is set to `false` in the command line, it simply skips the `enableNative` function call, but because the `useNative` variable is initialized with `true`, it never changes to `false`. There is also no other function available to set it to `false`. Because `useNative` can never be changed to `false` the native library implementations are always used.

I would suggest to initialize `useNative` with `false`. The native libraries would still be enabled by default, because in `BesuCommand` the `enableNative` functions will be called by default and set `useNative` to `true`. But if `--Xsecp256k1-native-enabled` or `--Xaltbn128-native-enabled` are set to `false` the `enableNative` functions will not be called and `useNative` stays `false` which is the intended behavior.

Please tell me your thoughts about the suggested change.

**Comment by RatanRSur** [comment]
Sounds good to me :+1:

**Comment by daniel-iobuilders** [comment]
@RatanRSur I have opened a PR to fix the issue #2163
